### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by sheepdestroyer** [frekky/iodine#1]
```
iodine.c: In function 'main':
iodine.c:741:3: warning: 'remote_forward_port' may be used uninitialized in this function [-Wmaybe-uninitialized]
   fprintf(stderr, "Requesting TCP data forwarding from server to %s:%d\n",
   ^
CC client.c
LD ../bin/iodine
CC iodined.c
iodined.c: In function 'main':
iodined.c:564:3: error: 'dns_fds' undeclared (first use in this function)
   dns_fds.v4fd = SD_LISTEN_FDS_START;
   ^
iodined.c:564:3: note: each undeclared identifier is reported only once for each function it appears in
Makefile:40: recipe for target 'iodined.o' failed
make[1]: *** [iodined.o] Error 1
make[1]: Leaving directory '/root/tmp/iodine-frekky/src'
Makefile:20: recipe for target 'all' failed
make: *** [all] Error 2
[root@vps193617 iodine-frekky]# 
```

**Comment by frekky** [comment]
Hi. Thanks for your interest, should be fixed in 8384da19.

Please let me know if it still doesn't compile for whatever reason.

**Comment by sheepdestroyer** [comment]
I don't remember if I managed to compile your branch at the time but I did try again fresh as of today on fedora 25 and got :
````[sheepdestroyer@sheepora iodine-frekky] $ make
make[1]: Entering directory '/home/sheepdestroyer/temp/iodine-frekky/src'
OS is LINUX, arch is x86_64
CC tun.c
CC dns.c
CC read.c
CC encoding.c
CC login.c
CC base32.c
CC base64.c
Making base64u.c
Making base64u.h
CC base64u.c
CC base128.c
CC md5.c
CC window.c
CC common.c
CC util.c
CC iodine.c
CC client.c
LD ../bin/iodine
CC iodined.c
iodined.c: In function ‘main’:
iodined.c:584:5: error: ‘dns_fds’ undeclared (first use in this function)
     dns_fds.v4fd = fd;
     ^~~~~~~
iodined.c:584:5: note: each undeclared identifier is reported only once for each function it appears in
Makefile:40: recipe for target 'iodined.o' failed
make[1]: *** [iodined.o] Error 1
````

**Comment by frekky** [comment]
At the moment I do not have a systemd compatible system to test with, however the issue should be fixed in b4f42ec82d50748324c3d6c82e00799a0811c425.

**Comment by sheepdestroyer** [comment]
Thank you, I can can confirm it now builds on both Arch and Fedora 25.
